Iryna_Ramanets/Module_1_Java/Java_Fundamentals/HW2/src/com/epam/cdp/chef/runner/Runner.java

```diff
@@ -0,0 +1,46 @@
+package com.epam.cdp.chef.runner;
+
+import com.epam.cdp.chef.interfaces.ICalculateCalories;
+import com.epam.cdp.chef.salad.Salad;
+import com.epam.cdp.chef.vegetables.Cucumber;
+import com.epam.cdp.chef.vegetables.Onion;
+import com.epam.cdp.chef.vegetables.Tomato;
+import com.epam.cdp.chef.vegetables.Vegetable;
+
+public class Runner {
+
+	private static void printCalories(ICalculateCalories calculatable) {
```

Comment:
I think you can add
System.out.print(message) to pribtCalories method. put message var to it